.pre-commit-config.yaml

```diff
@@ -5,39 +5,34 @@ default_stages: [commit]
 default_language_version:
     python: python3
 
+files: |
```

Comment:
This looks pretty good to me - lets land this before starting moving things.

Only gotcha - we have a few files in the root of the repo that should still get checked by default - can you doublecheck that those would be captured here ? e.g. the `.md` files, a few yaml and python files. 

At the very least the global trailing whitespace and end-of-file fixers and such should still run on those.

Reply:
I don't imagine files in the root are included.
Perhaps we should move Cobalt-specific ones
if we want to run our own checks on them.
Here's what it looks like we've added:
```
.codespellignorelines
.coveragerc
.pre-commit-config.yaml
.pylintrc
.style.yapf
BUILD_STATUS.md
CONTRIBUTING.md
docker-compose-windows-internal.yml
docker-compose-windows.yml
docker-compose.yml
download_resources.py
requirements.txt
setup.cfg
starboard_configuration.py
```

Reply:
There are a number of other files in the root that don't come from chromium, even though they might share a name with a file in chromium's root, i.e. `BUILD.gn` and `.gn`. At least for those two we'd want to include them for the gn format hook.

Reply:
I guess we can just include all files in the root.
PTAL

Reply:
Perhaps try a pattern that just includes all files in root - this would be something like `[^/]*` - e.g. any sequence that doesn't include forward slashes ?

Reply:
noticed you added this below, looks right

Reply:
LGTM